### PR TITLE
Use pip freeze for packages

### PR DESCRIPTION
**Pull request by Ladsgroup**
https://phabricator.wikimedia.org/T129110

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,27 @@
...
+mwtypes==0.2.0
+mwxml==0.2.1
+numpy==1.10.4
+ores==0.5.7
+python-Levenshtein==0.12.0
+pytz==2012c
+pywikibase==0.0.4
+redis==2.10.5
+requests==2.9.1
+revscoring==1.0.1
```
Comment (halfak) [inline requirements.txt]: Shouldn't be 1.0.1 yet.  This is due to an issue in editquality.  Switch this to 0.7.11 for now.

**Comment by halfak** [comment]
I don't see scipy or sklearn in the list.

File: requirements.txt [hunk 2]
```diff
@@ -1,7 +1,31 @@
...
+ores==0.5.7
+python-Levenshtein==0.12.0
+pytz==2012c
+pywikibase==0.0.4
+redis==2.10.5
+requests==2.9.1
+revscoring==0.7.11
+scikit-learn==0.17.1
+scipy==0.17.0
+sklearn==0.0
```
Comment (halfak) [inline requirements.txt]: 0.15.2

Comment (Ladsgroup) [inline requirements.txt]: returns error while making wheels. Let me check that more carefully

Comment (Ladsgroup) [inline requirements.txt]: with "scikit-learn==0.17.1" (line 22) it's not needed.

**Pull request merged**